### PR TITLE
Add Sak

### PR DESCRIPTION
**Pull request by eseiler**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by eseiler** [comment]
@bioconda/core

**Comment by bgruening** [comment]
@eseiler usually we compile all tools and not using precompiled binaries. Can you try that seqan is already available as dependency.

**Comment by eseiler** [comment]
@bgruening The current gcc versions don't support compiling the source (need at least >=4.9).
Also see
https://github.com/conda-forge/staged-recipes/pull/2089
and
https://github.com/bioconda/bioconda-recipes/issues/3224

**Comment by bgruening** [comment]
Hi Enrico,

fair enough then. This would be the correct PR to work further on this issue: https://github.com/conda-forge/staged-recipes/pull/873

File: recipes/sak/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+BINARY_HOME=$PREFIX/bin
+PKG_HOME=$PREFIX/opt/$PKG_NAME-$PKG_VERSION
```
Comment (bgruening) [inline recipes/sak/build.sh]: Is there any reason you put it under opt and not simply copy the binaries into PREFIX/bin?

**Comment by bgruening** [comment]
Thanks @eseiler!

**Pull request merged**